docs/user/security/role-mappings/index.asciidoc

```diff
@@ -2,11 +2,10 @@
 [[role-mappings]]
 === Role mappings
 
-Role mappings allow you to describe which roles to assign to your users
-using a set of rules. Role mappings are required when authenticating via
-an external identity provider, such as Active Directory, Kerberos, PKI, OIDC,
-or SAML.
+Role mappings are part of single sign-on (SSO), a https://www.elastic.co/subscriptions[subscription feature]. This feature allows you to
+describe which roles to assign to your users using a set of rules.
```

Comment:
I don't know how I feel about the new wording here. "Role mappings are part of single sign-on" sounds correct to me, but "Role mappings" itself is _a feature_. Thoughts @gchaps?

Reply:
I can't come up with anything better, so I'd leave it at feature.